xml/System.Net.Http/SocketsHttpConnectionContext.xml

```diff
@@ -55,8 +55,8 @@
         <ReturnType>System.Net.Http.HttpRequestMessage</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>The initial HttpRequestMessage that is causing the connection to be created.</summary>
+        <value>An HttpRequestMessage instance causing the connection to be created.</value>
         <remarks>To be added.</remarks>
```

Comment:
Why are the remarks empty?

Reply:
There were [no remarks in triple slash to port](https://github.com/dotnet/runtime/pull/42563/files), and the default string for empty remarks is `To be added. `